jcl/jpp_configuration.xml

```diff
@@ -244,19 +213,19 @@
 		  label="PANAMA"
 		  outputpath="PANAMA/src"
 		  flags="Panama"
-		  dependencies="JAVA14"
+		  dependencies="JAVA15"
```

Comment:
@tajila Are you aware of any uses of the PANAMA jcl config today?  If not, I won't worry about moving the base from 14 -> 15

Reply:
If it's not in use, perhaps it's time for it to be deleted?

Reply:
The Panama native-interop JDK has like changed substantially since we last supported it, I doubt it will work with the latest prototype. The PANAMA config can be deleted and the related MH code can be removed as well.

Reply:
Removing PANAMA seems to be a bigger piece of work: I opened https://github.com/eclipse/openj9/issues/10377 to track that.